Units/parser-gdscript.r/sample.d/input.gd

```diff
@@ -0,0 +1,79 @@
+# Taken from https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_basics.html
+
+# A file is a class!
+
+# Inheritance
+
+extends BaseClass
+
+# (optional) class definition with a custom icon
+
+class_name MyClass, "res://path/to/optional/icon.svg"
+
+
+# Member variables
+
+var a = 5
+var s = "Hello"
+var arr = [1, 2, 3]
+var dict = {"key": "value", 2: 3}
+var typed_var: int
+var inferred_type := "String"
+
+# Constants
+
+const ANSWER = 42
+const THE_NAME:String = "Charly"
+
+# Enums
+
+enum {UNIT_NEUTRAL, UNIT_ENEMY, UNIT_ALL}
+enum Named {THING_1, THING_2, ANOTHER_THING=1}
```

Comment:
It seems that your parser has the ability to tag THING_1, THING_2,...
I wonder why they are not in expected.tags.

Reply:
My first handwritten test case had lines for the enumerated values because I thought they'd show up. Thought this was how it was supposed to be, they do show up in the geany symbols list, back to checking code then. Local variables in functions also don't show up outside